netbox/utilities/views.py

```diff
@@ -88,15 +88,28 @@ def queryset_to_csv(self):
         Export the queryset of objects as comma-separated value (CSV), using the model's to_csv() method.
         """
         csv_data = []
+        custom_fields = []
 
         # Start with the column headers
         headers = ','.join(self.queryset.model.csv_headers)
+
+        # Add custom field headers
+        content_type = ContentType.objects.get_for_model(self.queryset.model)
```

Comment:
This should probably be wrapped by an `if` statement to clarify that the relevant logic applies only to models with custom fields.

You can use call `get_custom_fields()` on the model to return all applicable CustomFields.

Reply:
Can't use `get_custom_fields` here as it's specific to an instance of a model. Any alternatives?

Reply:
`self.queryset.model().get_custom_fields()`

We should really turn that into a class method, though.

Reply:
Thanks for that. I'll update the PR now.

Not sure about changing it to class method; the current implementation returns the custom fields' values of the current element, if any. Though to be honest, `get_custom_fields` is only used once (twice with this change), so changing it isn't that taxing.